src/style/content/history-export-import.less

```diff
@@ -15,31 +15,17 @@
     }
   }
 
-  .cancel-button {
-    .button-reset-default;
-
-    margin-bottom: 56px;
-    cursor: pointer;
-    font-size: @font-size-medium;
-    font-weight: @font-weight-regular;
-    line-height: @line-height-sm;
-    user-select: none;
-  }
-
   .history-message {
     margin: auto;
     text-align: center;
 
-    check-icon {
+    svg {
+      width: 32px;
+      height: 32px;
       margin-bottom: 16px;
 
-      svg {
-        width: 32px;
-        height: 32px;
-
-        path {
-          fill: @w-blue;
-        }
+      path {
+        fill: var(--success-color);
```

Comment:
Thanks @PatrykBuniX  :D 
Couldn't resist using it, checking with design now

(supposed to be blue, but was black for the past year I imagine)